envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/AccessLogStatusCodeFilterFactory.kt

```diff
@@ -0,0 +1,27 @@
+package pl.allegro.tech.servicemesh.envoycontrol.groups
+
+import com.google.re2j.Pattern
+import io.envoyproxy.envoy.config.filter.accesslog.v2.ComparisonFilter
+
+class AccessLogStatusCodeFilterFactory {
```

Comment:
Let's call this class `AccessLogFilterFactory` and move logic responsible for access log configuration from `EnvoyListenerFactory`.

Reply:
Class renamed and moved building status code filter to factory